components/aws/sagemaker/common/train.template.yaml

```diff
@@ -21,10 +21,13 @@ VpcConfig:
 StoppingCondition:
   MaxRuntimeInSeconds: 86400
   MaxWaitTimeInSeconds: 86400
+DebugHookConfig: {}
+DebugRuleConfigurations: []
+TensorBoardOutputConfig: {}
```

Comment:
what does the user experience for a KFP user look w.r.t this

Reply:
I asked Satadal about this and he said while he hasn't heard any customer feedback on TB, he thinks it would be better to add it to be feature parity with the debugger service.

Reply:
I am not sure if he is in sync with what KFP customers would expect. KFP has tensorboard integration and customers would expect the data to show up in that dashboard, we dont have that integration yet, lets discuss this offline. FYI @akartsky